pepys_import/core/store/data_store.py

```diff
@@ -616,7 +622,12 @@ def check_datafile(datafile):
             )
 
     def get_platform(
-        self, platform_name, pennant, nationality=None, platform_type=None, privacy=None
+        self,
```

Comment:
@BarisSari - I welcome your opinion on this, but...

The approach I've been taking in the API for measurements is that constructors only take the essential fields, and optional ones are provided by Setters.

Aah, but measurements are in-memory until they're submitted. 

For metadata, we make the database change immediately, I think.

Is this an argument for metadata constructors taking all their fields, like in this example?

Reply:
Yes, we make the database changes immediately for metadata tables. However, if you'd like to, we can ask it in missing data resolvers. So, this method doesn't take the pennant field.

Reply:
Agreed - let's remove the pennant field from the constructor.

Reply:
Removed:
https://github.com/debrief/pepys-import/pull/51/commits/0472158b6b068c246a70efb423a0c751eb716f9e